### PR TITLE
Fix quotes in command in bashlex_eval

### LINKED ISSUE #406: CIBW_ENVIRONMENT broken in 1.5.3
Somehow, my line of `export CIBW_ENVIRONMENT='... CCACHE_BASEDIR=`python -c "import tempfile; import os; print(os.path.realpath(tempfile.gettempdir()))"` ...'` (see https://github.com/YannickJadoul/Parselmouth/blob/1f3dd9abc63afdf4a1d26db40d210bb4187118a8/.travis.yml#L143; yes, I know I ought to clean this up) breaks:

```
    + /bin/true
    + mkdir -p /project
    + /opt/python/cp38-cp38/bin/python -c 'import sys, json, os; json.dump(os.environ.copy(), sys.stdout)'
    + uname -i
x86_64
    + python -c import 'tempfile;' import 'os;' 'print(os.path.realpath(tempfile.gettempdir()))'
  File "<string>", line 1
    import
```

See https://travis-ci.org/github/YannickJadoul/Parselmouth/jobs/709679114#L392

I'll start investigating, but is this perhaps related to #403, @joerick?

### PR DESCRIPTION
**Pull request by YannickJadoul**
Closes #406

### CODE REVIEW

File: cibuildwheel/bashlex_eval.py [hunk 1]
```diff
@@ -1,15 +1,14 @@
-import shlex
 import subprocess
 from typing import Callable, Dict, List, NamedTuple, Optional, Sequence
 
 import bashlex  # type: ignore
 
 # a function that takes a shell command and the environment, and returns the result
```
Comment (joerick) [inline cibuildwheel/bashlex_eval.py:6]:
```suggestion
# a function that takes a command and the environment, and returns the result
```

**Comment by joerick** [comment]
I'm gonna get this merged and another release out

**Comment by YannickJadoul** [comment]
Great, thanks! :-)

All in all just a debugging run, once I had the example of what goes wrong. But yes, it was confusing enough. I first thought replacing `node.word` with `context.input[part.pos[0]:part.pos[1]]` would be enough, but then the tests showed me that gives another problem (the one you had actually just solved), so .. yes, I noticed :-D

**Comment by joerick** [comment]
This kind of code is so subtle that it's better defined by the tests than the code itself, I think. anywho, released as v1.5.4 :)

**Comment by YannickJadoul** [comment]
Hurray! (Almost too quick to have had pinned cibuildwheel==1.5.2 ;-) But anyway, reverting it: https://github.com/YannickJadoul/Parselmouth/commit/a1e674b43c76d31ef2187bc6017045ccb5fa2793)

**Pull request merged**